packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1374,7 +1399,7 @@ abstract class BuildableElement extends Element {
 typedef Widget WidgetBuilder(BuildContext context);
 typedef Widget IndexedBuilder(BuildContext context, int index);
 
-// See _builder.
+// See ComponentElement._builder.
```

Comment:
If you make it a `///` dartdoc you can make `[ComponentElement._builder]` linkable.